contents/select.md

```diff
@@ -0,0 +1,191 @@
+## Select {#sec:select}
+
+Whereas `filter` removes rows, `select` removes columns.
+However, `select` is much more versatile than just removing columns, as we will discuss in this section.
+First, let's create a dataset with multiple columns:
+
+```jl
+@sco responses()
+```
+
+Here, the data represents answers on five questions (`q1`, `q2`, ..., `q5`) in a questionnaire.
```

Comment:
```suggestion
Here, the data represents answers for five questions (`q1`, `q2`, ..., `q5`) in a questionnaire.
```

I think is "answers for" not "answers on". But my grammar even in Portuguese sucks...